### ISSUE TITLE
[Package Issue]: Ryujinx.Ryujinx

### ISSUE THREAD

**Issue by Clockwork-Muse** [microsoft/winget-pkgs#176342]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

As with Switch emulator Yuzu, today the Switch emulator Ryujinx [has also been taken offline](https://www.polygon.com/gaming/458905/nintendo-switch-emulator-ryujinx-offline).

The relevant GitHub org/project pages already 404, so packages will no longer install.

### Steps to reproduce

Attempt to install Ryujinx

### Actual behavior

Will not install

### Expected behavior

N/A

### Environment

```raw
N/A
```

**Comment by Clockwork-Muse** [comment]
@Dragon1573 - Apparently there's an outstanding manifest for 1.1.1403 that's still present.

Also, there's an additional (I think "canary") package - `Ryujinx.Ryujinx.Ava` - that's part of the same org.  Should I open a new issue for that?

**Comment by Dragon1573** [comment]
Now all removal request are approved by moderators. But it's still waiting for being collected and published to the "source endpoint" (it's somewhere else than the repository).

Please be patient. ❤️

**Comment by Dragon1573** [comment]
> Also, there's an additional (I think "canary") package - `Ryujinx.Ryujinx.Ava` - that's part of the same org. Should I open a new issue for that?

If this organization is totally blocked by someone and unable to provide any packages to normal users, it would be better creating another issue for asking manifests removal.
